packages/devtools-reps/src/object-inspector/tests/test-utils.js

```diff
@@ -34,7 +34,12 @@ function formatObjectInspector(wrapper: Object) {
       } else {
         arrowStr = "  ";
       }
-      return `${indentStr}${arrowStr}${getSanitizedNodeText(node)}`;
+
+      let icon = "";
+      if (node.find(".node").first().hasClass("block")) {
```

Comment:
maybe this can be a ternary? Unless there is a chance that this will have further types...

```javascript
const icon = node.find(".node").first().hasClass("block")
  ? "☲ "
  : "";
```

Reply:
✅